components/calendar/src/date.rs

```diff
@@ -353,23 +353,23 @@ impl<A: AsCalendar> Date<A> {
     ///
```

Comment:
do the docs need updating as well? this still talks about wrapping a calendar type, and the second sentence only makes sense if it explains why a `Date<Rc<AnyCalendar>>` is useful

Reply:
https://github.com/unicode-org/icu4x/pull/6407